src/components/Camera/style.css

```diff
@@ -132,3 +131,36 @@
     }
   }
 }
+
+.errorContainer {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+
+  @media (--small-viewport) {
+    top: 40px;
+  }
+
+  &::before {
+    content: '';
+    display: block;
+    height: 100vh;
```

Comment:
To prevent users from clicking anywhere else on the page:

> user should not be able to click on any other part of the screen apart from the retry link

Reply:
I see, this is for the mobile screen, right?

Reply:
Yes @stefaniacardenas pointed out a few days ago in a similar PR https://github.com/onfido/onfido-sdk-ui/pull/349#issuecomment-399902699

> All actions should sit at the bottom of the box on mobile